e3/testsuite/__init__.py

```diff
@@ -18,6 +18,11 @@
 import re
 import tempfile
 
+try:
+    from lxml import etree
+except ImportError:
+    import xml.etree.ElementTree as etree
```

Comment:
Any advantage in trying to use lxml here ? If modules from standard library are enough I would just use them

Reply:
Nothing relevant, I’ll remove it.